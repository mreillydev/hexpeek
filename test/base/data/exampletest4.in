10 offset
,5v
