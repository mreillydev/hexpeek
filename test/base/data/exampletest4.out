10
0000000000000010: 00 000 000 00000000 ././0 '\0'
0000000000000011: 11 017 021 00010001 4/0/2 DC1
0000000000000012: 22 034 042 00100010 5/1/2 '"'
0000000000000013: 33 051 063 00110011 5/0/4 '3'
0000000000000014: 44 068 104 01000100 6/2/2 'D'
